mmv1/products/netapp/backup.yaml

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 Google Inc.
```

Comment:
Could you rename this file to `Backup.yaml`? We generally use PascalCase for resource filenames.

Reply:
Done